web/src/settings_users.js

```diff
@@ -260,13 +260,7 @@ function human_info(person) {
     info.is_current_user = people.is_my_user_id(person.user_id);
     info.cannot_deactivate = info.is_current_user || (person.is_owner && !current_user.is_owner);
     info.display_email = person.delivery_email;
-
-    if (info.is_active) {
-        // TODO: We might just want to show this
-        // for deactivated users, too, even though
-        // it might usually just be undefined.
-        info.last_active_date = get_last_active(person);
-    }
```

Comment:
I am not sure we should remove this comment as we still do not show the last active date for deactivated users. Might be useful to keep it but somewhere else like where we set the `display_last_active_in_table` as false for deactivated users.

Reply:
@sahil839 Although, we aren't showing the `last_active_date` for **deactivated users** in deactivated users table but we are still showing it in users table.

Reply:
Yeah, but I guess the comment was added considering we might want to add last active column in the "Deactivated users" table as well as that function is called for both tables, but I am not sure.